de_templates/ingest/Ingest_SourceType_SourceName/data_factory/adf_triggers.tf.jinja

```diff
@@ -0,0 +1,24 @@
+{% set pipeline_name = 'Ingest_' ~ dataset_name -%}
+resource "azurerm_data_factory_trigger_tumbling_window" "trigger_{{ pipeline_name }}" {
+  name            = "trg_{{ pipeline_name }}"
+  data_factory_id = data.azurerm_data_factory.factory.id
+  start_time      = "2010-01-01T00:00:00Z"
+  end_time        = "2011-12-31T23:59:59Z"
+  frequency       = "{{ trigger_frequency }}"
+  interval        = {{ trigger_interval }}
+  delay           = "02:00:00"
+  max_concurrency = 4
+  activated       = false
```

Comment:
same as for the main pipeline - I think the trigger settings should go to vars

Reply:
Although I'm not really sure about it, so I'll resolve. I think both options have pros and cons :)